### PR TITLE
Settings page design changes

### PR DESCRIPTION
**Pull request by mikeshultz**
### Description:

Design changes to the settings page as directed by @micahalcorn and Aure.

This removes Disable button for browser notifiactions, and hides service endpoints behind a "Developer Mode".  The UI has been implemented for Disable Messaging and Decentralized Mode, but is currently hidden until the underlying functionality is done(separate PRs and issues).

settings_page-20180524-01

### Checklist:

- [x] Test your work and double-check to confirm that you didn't break anything
- [x] [Wrap any new text/strings with `fbt`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#wrapping-text) for translation
- [x] If there are any new/changed strings to translate, [`npm run translate`](https://github.com/OriginProtocol/origin/tree/master/dapps/marketplace/translation#integrating-translations)
- [ ] Update any relevant READMEs and [docs](https://github.com/OriginProtocol/origin/tree/master/docs)

Ref: #2333

### CODE REVIEW

File: dapps/marketplace/src/components/Toggle.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+  handleClick() {
+    const newToggled = !this.state.isToggledOn
+    this.setState({
+      isToggledOn: newToggled
+    })
+    console.debug('toggle clicked! toggled? ', newToggled)
```
Comment (nick) [inline dapps/marketplace/src/components/Toggle.js]: Can probably remove this

File: dapps/marketplace/src/pages/settings/Settings.js [hunk 2]
```diff
@@ -73,10 +79,16 @@ class Settings extends Component {
     )
   }
 
+  toggleDeveloperMode(on) {
+    console.debug(`toggleDeveloperMode(${on})`)
```
Comment (nick) [inline dapps/marketplace/src/pages/settings/Settings.js]: Remove debug :)

**Comment by nick** [comment]
Looks good, just one thing to fix on mobile:

**Comment by mikeshultz** [comment]
hm, didn't see this when testing with devtools' mobile emulation.  Do you know what the screen size is for your device, @nick?

**Comment by mikeshultz** [comment]
Nevermind, am able to repro

**Comment by mikeshultz** [comment]
Fixed to properly use bootstrap column stacking.  Thanks for catching that.

**Pull request merged**